src/ServiceControl.Monitoring/QueueLength/QueueLengthDataStore.cs

```diff
@@ -2,13 +2,11 @@ namespace ServiceControl.Monitoring.QueueLength
 {
     using System.Collections.Generic;
     using System.Linq;
-    using Http;
     using Newtonsoft.Json.Linq;
-    using Raw;
 
-    class QueueLengthDataConsumer : IRawDataConsumer, IEndpointDataProvider
+    public class QueueLengthDataStore
```

Comment:
This "store" and the others (critical/processing time) seem to operate on different levels of abstraction. This one wants json and extracts the info it needs. The others want values already extracted from the message. I'm not sure it really matters but I thought I'd point it out.